### PR TITLE
Increase coverage chsdi.views.shortener

### PR DESCRIPTION
**Pull request by mariavkourouni**

### CODE REVIEW

**Comment by mariavkourouni** [comment]
jenkins test

File: chsdi/tests/integration/test_shortener.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+ # -*- coding: utf-8 -*-
+
+from chsdi.tests.integration import TestsBase
+
+
+class TestShortenerView(TestsBase):
+
+    def test_shortener_toolong_url(self):
+        test_url = 'https://map.geo.admin.ch/?topic=ech&lang=en&bgLayer=ch.swisstopo.pixelkarte-farbe' + \
```
Comment (loicgasser) [inline chsdi/tests/integration/test_shortener.py]: `+` and `\` are redundant here. use only `\`

File: chsdi/tests/integration/test_shortener.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            '%20Cropland%7C%7Chttps:%2F%2Fwms.geo.admin.ch%2F%7C%7Cch.blw.bodeneignung-kulturland,' + \
+            'ch.swisstopo.pixelkarte-farbe-pk1000.noscale,ch.swisstopo.pixelkarte-farbe-pk500.noscale,' + \
+            'ch.swisstopo.pixelkarte-farbe-pk200.noscale,ch.swisstopo.pixelkarte-farbe-pk100.noscale,' + \
+            'ch.swisstopo.pixelkarte-pk50.metadata&layers_timestamp=19961231,,,,,,,,,,,,,,,,,,,,,' + \
+            '&X=172839.76&Y=662412.05&zoom=3&time=1996&layers_opacity=0.25,1,1,0.4,1,1,1,1,1,1,1,1,1,1,1,' + \
+            '0.55,0.45,1,1,1,1,1&catalogNodes=457,458'
+        self.testapp.get('/shorten.json', params={'url': test_url}, status=200)
+
+    def test_shortener_redirect(self):
+        test_shortenurl = 'http://api3.geo.admin.ch/shorten.json?url=https:%2F%2Fmf-geoadmin3.int.bgdi.ch' + \
```
Comment (loicgasser) [inline chsdi/tests/integration/test_shortener.py]: We should not test prod here and it is not a redirection...

Comment (loicgasser) [inline chsdi/tests/integration/test_shortener.py]: what you want to test instead is `shortent/6863fbb96f` for instance. We expect a 301 and given url.

**Pull request merged**